src/App.tsx

```diff
@@ -15,36 +15,87 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  None = '',
+  Alphabetically = 'alphabetically',
+  Length = 'length',
+}
+
 export const App: React.FC = () => {
+  const [goods, setGoods] = useState<string[]>([...goodsFromServer]);
+  const [sortType, setSortType] = useState<SortType>(SortType.None);
+  const [isReversed, setIsReversed] = useState<boolean>(false);
+
+  const sortGoods = (type: SortType) => {
```

Comment:
[CODE STYLE] - The function's name 'sortGoods' does not start with a verb that clearly indicates what the function does. Consider renaming it to something like 'handleSortGoods' to improve readability and indicate that it is an event handler.